src/components/Input/SearchInput.module.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../styles/theme';
+
+@mixin path($color, $styles) {
+    svg {
+        path {
+            stroke: $color;
+        }
+    }
+}
+
+.SearchWrapper {
+    border-radius: 1000px !important; // make it pill shaped
+}
+
+.SearchIcon {
+    &[data-is-searching] {
```

Comment:
so here `data-is-searching` is a boolean value, and the css is picked on the basis of it being `true`/`false`?

Reply:
Yup! It's basically saying `if data-is-searching has a value`, and in the component we set `data-is-searching={isSearching ? '' : null}` ☺️

Reply:
got it, thanks